runelite-client/src/main/java/net/runelite/client/plugins/poh/IncenseBurner.java

```diff
@@ -25,20 +25,46 @@
 package net.runelite.client.plugins.poh;
 
 import java.time.Instant;
-import lombok.AllArgsConstructor;
+import javax.inject.Inject;
 import lombok.Getter;
-import lombok.RequiredArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
-@RequiredArgsConstructor
-@AllArgsConstructor
 class IncenseBurner
 {
+	@Inject
+	private PohConfig config;
+
```

Comment:
This is unused

Reply:
Oops, it is not used correctly as it should have been.